plugins/localization-import-export/src/App.tsx

```diff
@@ -84,30 +84,32 @@ export function App() {
 
     return (
         <main>
-            <div className="asset">
-                <svg xmlns="http://www.w3.org/2000/svg" width="74" height="74">
-                    <path
-                        d="M 37 2.313 C 56.157 2.313 71.688 17.843 71.688 37 C 71.688 56.157 56.157 71.688 37 71.688 C 17.843 71.688 2.313 56.157 2.313 37 C 2.313 17.843 17.843 2.313 37 2.313 Z M 11.563 37 C 11.563 47.692 18.159 56.843 27.504 60.606 C 24.809 54.7 23.125 46.309 23.125 37 C 23.125 27.691 24.809 19.3 27.504 13.394 C 18.159 17.157 11.563 26.308 11.563 37 Z M 62.438 37 C 62.438 26.308 55.841 17.157 46.496 13.394 C 49.191 19.3 50.875 27.691 50.875 37 C 50.875 46.309 49.191 54.7 46.496 60.606 C 55.841 56.843 62.438 47.692 62.438 37 Z M 30.063 37 C 30.063 51.049 33.169 62.438 37 62.438 C 40.831 62.438 43.938 51.049 43.938 37 C 43.938 22.951 40.831 11.563 37 11.563 C 33.169 11.563 30.063 22.951 30.063 37 Z"
-                        fill="currentColor"
-                    ></path>
-                    <path
-                        d="M 6.937 34.688 L 6.938 34.688 C 26.719 39.253 47.281 39.253 67.063 34.688 L 67.063 34.688"
-                        fill="transparent"
-                        strokeWidth="7"
-                        stroke="currentColor"
-                        stroke-linecap="round"
-                        stroke-linejoin="round"
-                    ></path>
-                </svg>
+            <div className="intro">
+                <div className="asset">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 74 74">
+                        <path
+                            d="M 37 2.313 C 56.157 2.313 71.688 17.843 71.688 37 C 71.688 56.157 56.157 71.688 37 71.688 C 17.843 71.688 2.313 56.157 2.313 37 C 2.313 17.843 17.843 2.313 37 2.313 Z M 11.563 37 C 11.563 47.692 18.159 56.843 27.504 60.606 C 24.809 54.7 23.125 46.309 23.125 37 C 23.125 27.691 24.809 19.3 27.504 13.394 C 18.159 17.157 11.563 26.308 11.563 37 Z M 62.438 37 C 62.438 26.308 55.841 17.157 46.496 13.394 C 49.191 19.3 50.875 27.691 50.875 37 C 50.875 46.309 49.191 54.7 46.496 60.606 C 55.841 56.843 62.438 47.692 62.438 37 Z M 30.063 37 C 30.063 51.049 33.169 62.438 37 62.438 C 40.831 62.438 43.938 51.049 43.938 37 C 43.938 22.951 40.831 11.563 37 11.563 C 33.169 11.563 30.063 22.951 30.063 37 Z"
+                            fill="currentColor"
+                        ></path>
+                        <path
+                            d="M 6.937 34.688 L 6.938 34.688 C 26.719 39.253 47.281 39.253 67.063 34.688 L 67.063 34.688"
+                            fill="transparent"
+                            strokeWidth="7"
+                            stroke="currentColor"
+                            stroke-linecap="round"
+                            stroke-linejoin="round"
+                        ></path>
+                    </svg>
```

Comment:
Nit, inline SVG always looks scary. I would move it into a component with a descriptive name